integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -174,32 +169,32 @@ public static void init(@Namespaces(3) List<String> namespaces) {
     // install and verify Kibana
     logger.info("install and verify Kibana");
     kibanaParams = assertDoesNotThrow(() -> installAndVerifyKibana(elasticSearchNs),
-        String.format("Failed to install Kibana"));
+            String.format("Failed to install Kibana"));
     assertNotNull(kibanaParams, "Failed to install Kibana");
 
     // install and verify Operator
     installAndVerifyOperator(opNamespace, opNamespace + "-sa",
-        false, 0, true, domainNamespace);
+            false, 0, true, domainNamespace);
 
     elasticSearchHost = "elasticsearch." + elasticSearchNs + ".svc.cluster.local";
 
     // upgrade to latest operator
     HelmParams upgradeHelmParams = new HelmParams()
-        .releaseName(OPERATOR_RELEASE_NAME)
-        .namespace(opNamespace)
-        .chartDir(OPERATOR_CHART_DIR);
+            .releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
 
     // build operator chart values
     OperatorParams opParams = new OperatorParams()
-        .helmParams(upgradeHelmParams)
-        .elkIntegrationEnabled(true)
-        .elasticSearchHost(elasticSearchHost);
+            .helmParams(upgradeHelmParams)
+            .elkIntegrationEnabled(true)
+            .elasticSearchHost(elasticSearchHost);
 
     assertTrue(upgradeAndVerifyOperator(opNamespace, opParams),
-        String.format("Failed to upgrade operator in namespace %s", opNamespace));
+            String.format("Failed to upgrade operator in namespace %s", opNamespace));
 
     // create fluentd configuration
-    configFluentd();
+    //configFluentd();
```

Comment:
Remove commented out code and the preceding comment.

Reply:
fixed